app/src/main/res/values/strings.xml

```diff
@@ -1268,6 +1268,17 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_smoothness_obstacle_hint">Please provide a general answer for the smoothness. Single obstacles like kerbs, speed bumps or cattle grids can be ignored. Repeated or unexpected obstacles should count towards the answer.</string>
     <string name="quest_smoothness_hint">🚲, 🚗, 🚙, … indicate for which vehicles the surface should be okay to use. What matters is the big picture rather than single damages.</string>
 
+    <string name="quest_building_entrace_title">What kind of building entrance is this?</string>
+    <string name="quest_building_entrance_main">Main (building may have multiple main entrances)</string>
+    <string name="quest_building_entrance_staircase">Staircase</string>
+    <string name="quest_building_entrance_service">Service</string>
+    <string name="quest_building_entrance_exit">Exit only</string>
+    <string name="quest_building_entrance_emergency_exit">Emergency exit only</string>
+    <string name="quest_building_entrance_garage">Garage</string>
+    <string name="quest_building_entrance_shop">Shop (storefront entrance), not a main building entrance</string>
+    <string name="quest_building_entrance_yes">Other entrance type</string>
+    <string name="quest_building_entrance_dead_end">Dead end, no entrance</string>
```

Comment:
Regarding wording:

- doesn't mention that "main" should be used on public buildings etc. according to wiki
- hint that a building may have several main entrances seems superfluous because unlike in JOSM etc. the user doesn't see which other entrances are tagged as "main"
- I suggest "Staircase entrance (e.g. of apartment building)" because it is mostly/supposed to be used on apartment buildings
- I think for some values, "entrance" needs to be added because otherwise it is not well understood or doesn't make sense. E.g. "service" is not really translatable in German. Even German word for "service entrance" would be literally translated "servant entrance" which is itself a outdated term. I think the modern term would be "Nebeneingang" ("side entrance").
- "Garage": What does this mean, is this an entrance into a car garage? Or literally the garage door of a garage? (Why would anyone tag this? And why would a **footway** lead into a car garage rather than a `highway=service`?) 
- "other entrance type": Wording implies that the user read through all other options. Maybe instead just "unspecific entrance"

Reply:
> Wording implies that the user read through all other options

That is intentional. 

> Unspecific entrance

I got convinced by native speakers that "other" is better :) https://osmus.slack.com/archives/C01BU04GW7L/p1648903729432579

> Garage

removed

> I think for some values, "entrance" needs to be added because otherwise it is not well understood or doesn't make sense

done, for all (for consistency)

Other hints also followed

Current state:

```
    <string name="quest_building_entrace_title">What kind of building entrance is this?</string>
    <string name="quest_building_entrance_main">Main entrance on a public building</string>
    <string name="quest_building_entrance_staircase">Staircase entrance (e.g. of apartment building)</string>
    <string name="quest_building_entrance_service">Service entrance</string>
    <string name="quest_building_entrance_exit">Exit only</string>
    <string name="quest_building_entrance_emergency_exit">Emergency exit only</string>
    <string name="quest_building_entrance_shop">Shop entrance (storefront entrance), not a main building entrance</string>
    <string name="quest_building_entrance_yes">Other entrance type</string>
    <string name="quest_building_entrance_dead_end">Dead end, no entrance</string>
```